testing/cucumber.yml

```diff
@@ -1,7 +1,9 @@
 passing: --tags 'not @failing and not @future_release'
 
-html: --format html --out artifacts/reports/report.html
-junit: --format junit --out artifacts/reports
-json: --format json --out artifacts/reports/report.json
+<% base_path = "#{ENV['BROWSER']}/#{ENV.fetch('BROWSERSTACK_CONFIGURATION_OPTIONS', 'grid')}" %>
```

Comment:
this wants to be evaluated <%=

Reply:
Does it? It's setting a variable which is then used in the profile statements, so it shouldn't be outputting anything on that line

Reply:
Hmm ok, maybe I'm wrong here. Because it's doing both evaluation and setting. Maybe not then.